docs/docs/illixr_plugins.md

```diff
@@ -2,290 +2,641 @@
+## record_rgb_depth
+
+Writes [`rgb_depth_type`][62] data to disk.
+
+Topic details:
+
+-   Synchronously *reads* [`rgb_depth_type`][62] from `rgb_depth` topic.
+
+**Details** [**Code**][C25]
+
+## rk4_integrator
+
+Integrates over all [_IMU_][36] samples since the last published [_SLAM_][39] [_pose_][37] to
+provide a [_fast pose_][37] every time a new IMU sample arrives using RK4 integration.
+
+Topic details:
+
+-   Asynchronously *reads* [`imu_integrator_input`][66] from `imu_integrator_input` topic.
+-   Synchronously *reads* [`imu_type`][64] from `imu` topic.
+-   *Publishes* [`imu_raw_type`][65] to `imu_raw` topic.
 
--   [`hand_tracking_gpu`][43]:
-    Detects and identifies hands in an image, GPU based calculations. The output from this plugin can be used to track hand movements and recognize hand gestures.
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C26]
 
-    Topic details:
+## tcp_network_backend
 
-    -   Synchronously *reads/subscribes* to one of `frame_type` on `webcam` topic, `binocular_cam_type` on `cam` topic, or `cam_type_zed` on `cam_zed` topic. This is selectable at run time via an environment variable.
-    -   *Publishes* `ht_frame` on `ht` topic.
+Provides network communications over TCP.
 
--   [`hand_tracking.viewer`][43]:
-    Reads the output of the `hand_tracking` plugin and displays the results on the screen. This is most useful for debugging. The capabilites of this plugin will be merged into the `debugview` plugin in the future.
+**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C27]
 
-    Topic details:
+## timewarp_gl [^1]
 
-    -   Synchronously *reads/subscribes* to `ht_frame` on `ht` topic.
+[Asynchronous reprojection][35] of the [_eye buffers_][34].
+The timewarp ends just after [_vsync_][34], so it can deduce when the next vsync will be.
 
--   [`webcam`][44]:
-    Uses a webcam to capture images for input into the `hand_tracking` plugin. This plugin is useful for debugging and is not meant to be used in a production pipeline.
+Topic details:
 
-    Topic details:
+-   *Calls* [`pose_prediction`][57]
+-   *Publishes* `hologram_input` to `hologram_in` topic.
+-   If using Monado
+    - Asynchronously *reads* `rendered_frame` on `eyebuffer` topic, if using Monado.
+    - *Publishes* `time_point` to `vsync_estimate` topic.
+    - *Publishes* `texture_pose` to `texture_pose` topic if `ILLIXR_OFFLOAD_ENABLE` is set in the env.
+-   If *not* using Monado
+    - *Publishes*  `signal_to_quad` to `signal_quad` topic.
 
-    -   *Publishes* `monocular_cam_type` on `webcam` topic.
+&nbsp;&nbsp;[**Details**][D28]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C28]
 
--   [`hologram`][9]:
-    Adapts the eyebuffer for use on a holographic display.
-    By default, this plugin is disabled, since an NVIDIA GPU is currently required.
+## timewarp_vk
 
-    Topic details:
+[Asynchronous reprojection][35] of the [_eye buffers_][34].
+The timewarp ends just after [_vsync_][34], so it can deduce when the next vsync will be.
```

Comment:
Why "just after"? It should be "right before"

Reply:
fixed